src/Organizations/OrganizationCreate/OrganizationCreate.js

```diff
@@ -32,7 +32,7 @@ const OrganizationCreate = ({ history, location, mutator, stripes, intl }) => {
   const cancelForm = useCallback(
     (id) => {
       history.push({
-        pathname: id ? `/organizations/${id}/view` : '/organizations',
+        pathname: id ? `/organizations/view/${id}` : '/organizations',
```

Comment:
maybe it makes sense to create constant for '/organizations/view' route?

Reply:
added a constant